src/views/Pools/index.tsx

```diff
@@ -195,7 +196,18 @@ const Pools: React.FC = () => {
       case 'totalStaked':
         return orderBy(
           poolsToSort,
-          (pool: Pool) => (pool.isAutoVault ? totalCakeInVault.toNumber() : pool.totalStaked.toNumber()),
+          (pool: Pool) => {
+            let totalStaked
+            if (pool.isAutoVault) {
+              totalStaked = getBalanceNumber(totalCakeInVault, pool.stakingToken.decimals)
+            } else if (pool.sousId === 0) {
+              const manualCakeTotalMinusAutoVault = new BigNumber(pool.totalStaked).minus(totalCakeInVault)
```

Comment:
We are trying to deprecate BigNumber.js. Can you rewrite this to use ether's BigNumber.

Reply:
Done